egs/vcc2020/csmsc_mailabs/local/text/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Keith Ito
```

Comment:
It seems that while `vcc2020/tts1/local/text` is a symbolic link, the others are copied one.
To avoid too many duplicated scripts, could you change all of `local/text` to a symbolic link?